metron-interface/metron-alerts/e2e/alerts-list/alerts-list.e2e-spec.ts

```diff
@@ -0,0 +1,119 @@
+/// <reference path="../matchers/custom-matchers.d.ts"/>
```

Comment:
We are missing the Apache license header in this Typescript file and some others.